scripts/yum-wrapper

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Installing additional packages is not supported. Continue [y/n]"
```

Comment:
I hope we can refer to a document or a wiki page where we describe the issue in more detail and provide possible alternatives.

I also believe it should be more prominent...